xml/Microsoft.VisualBasic.Logging/FileLogTraceListener.xml

```diff
@@ -227,14 +227,14 @@
         <ReturnType>System.Boolean</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>Determines whether to append the output to the current file or write it to a new file.</summary>
+        <summary>Determines whether to append the output to the current file or write it to a new or existing file.</summary>
         <value>
           <see langword="Boolean" />, with <see langword="True" /> indicating that the output is appended to the current file, and <see langword="False" /> indicating that output is written to a new file. The default setting for this property is <see langword="True" />.</value>
```

Comment:
The \<value> section needs to be changed as well. It's also awkward (the Boolean is repeated from the line above). Better would be:

> \<see langword="True" /> to append the output to the current file, and \<see langword="False" /> to write the output to a new or existing file. The default setting...

Reply:
Maybe: 

> <see langword="True" /> to append the output to the current file.  If the file does not exist, <see langword="False" /> writes output to a new file. If the file already exists, <see langword="False" /> deletes the previous contents of the file before writing the output. The default setting..